output/tables/demographics.csv

```diff
@@ -1,5362 +1,5396 @@
-UY,3449299,1665268,1784031,160944,3288355,1736989,1736989,19.708028,0.804
-UZ,32955400,16432433,16522967,16320173,16635227,2463969,2463969,77.469205,0.71
-VA,1000,,,,,,,,
-VC,110210,55977,54233,52683,57527,,,282.58974,0.723
-VE,28870195,14252274,14617921,3404373,25465822,2934560,9339955,32.730792,0.761
-VE_B,1440875,,,,,,,,
-VE_C,587056,,,,,,,,
-VE_D,1629433,,,,,,,,
-VE_F,2069064,,,,,,,,
-VE_G,2106264,,,,,,,,
-VE_H,500420,,,,,,,,
-VE_I,902847,,,,,,,,
-VE_J,964264,,,,,,,,
-VE_N,905443,,,,,,,,
-VE_O,426337,,,,,,,,
-VE_U,499049,,,,,,,,
-VE_V,4323467,,,,,,,,
-VE_Y,167676,,,,,,,,
-VE_Z,1422000,,,,,,,,
-VG,29802,,,15580,14222,,,198.68,
-VI,106977,50881,56096,4578,102399,,,305.64857,
-VN,95540395,47678087,47862308,61223241,34317154,8145361,16156245,308.12525,0.694
-VU,292680,148422,144258,218708,73972,,,24.009844,0.603
-WF,12197,,,,,,,,
-WS,196130,101485,94645,160350,35780,,,69.303887,0.713
-XK,1883018,,,,,,,,
-YE,28498687,14360016,14138671,18056198,10442489,2779317,2779317,53.977853,0.452
-YT,256518,,,,,,,,
-ZA,57779622,28488734,29290888,19439954,38339668,5485986,20400727,47.63012,0.699
-ZA_EC,6436761,,,,,,,,
-ZA_FS,2753200,,,,,,,,
-ZA_GP,12272263,,,,,,,,
-ZA_KZN,10267300,,,,,,,,
-ZA_LP,5404868,,,,,,,,
-ZA_MP,2065883,,,,,,,,
-ZA_NC,1145861,,,,,,,,
-ZA_NW,3509953,,,,,,,,
-ZA_WC,6844272,,,,,,,,
-ZM,17351822,8590330,8761492,9800136,7551686,2523844,2523844,23.341479,0.588
-ZW,14439018,6879217,7559801,9788355,4650663,1515016,1515016,37.324591,0.535
+key,population,population_male,population_female,population_rural,population_urban,population_largest_city,population_clustered,population_density,human_development_index,population_age_00_09,population_age_10_19,population_age_20_29,population_age_30_39,population_age_40_49,population_age_50_59,population_age_60_69,population_age_70_79,population_age_80_89,population_age_90_99
+AD,77006,58614,55571,9193,67813,,,163.84255,0.858,0,3112,2703,3644,5609,5248,3329,1812,1805,451
+AE,9630959,6680352,2950607,1298061,8332898,2785376,5775924,135.60911,0.863,0,188936,942828,929892,655942,268297,38111,15101,6036,1509
+AF,37172386,19093281,18079105,27695286,9477100,4011770,4011770,56.93776,0.498,0,1874110,1481822,1020489,564723,344061,193498,70707,36258,9065
+AF_BAM,492400,,,,,,,,,,,,,,,,,,
+AF_BDG,500000,,,,,,,,,,,,,,,,,,
+AF_BDS,823000,,,,,,,,,,,,,,,,,,
+AF_DAY,417300,,,,,,,,,,,,,,,,,,
+AF_GHO,680200,,,,,,,,,,,,,,,,,,
+AF_HEL,859600,,,,,,,,,,,,,,,,,,
+AF_HER,1762157,,,,,,,,,,,,,,,,,,
+AF_KAB,4012000,,,,,,,,,,,,,,,,,,
+AF_KAP,399500,,,,,,,,,,,,,,,,,,
+AF_KDZ,820000,,,,,,,,,,,,,,,,,,
+AF_KNR,578500,,,,,,,,,,,,,,,,,,
+AF_LAG,4035000,,,,,,,,,,,,,,,,,,
+AF_NIM,149000,,,,,,,,,,,,,,,,,,
+AF_PAR,664502,,,,,,,,,,,,,,,,,,
+AF_PIA,4992000,,,,,,,,,,,,,,,,,,
+AF_PKA,393800,,,,,,,,,,,,,,,,,,
+AF_SAR,532000,,,,,,,,,,,,,,,,,,
+AF_WAR,540100,,,,,,,,,,,,,,,,,,
+AG,96286,46421,49865,72601,23685,,,218.83182,0.78,0,3878,3652,3166,3127,2620,1447,540,441,110
+AI,16086,7150,7971,,,,,,,0,545,470,456,490,452,268,130,107,27
```

Comment:
Here, population_male + population_female < population.  Is this just because of using different data sources?

Reply:
Yes, see comment above. Based on the pipeline configuration, the values for any column could be overridden by `Wikidata` or `WorldBank`.